datastore/src/main/java/com/google/datastore/snippets/Concepts.java

```diff
@@ -114,12 +109,7 @@ public static void beforeClass() throws IOException, InterruptedException {
    */
   @Before
   public void setUp() {
-    datastore = DatastoreOptions.builder()
-        .projectId(PROJECT_ID)
-        .namespace("ghijklmnop")
-        .host("http://localhost:" + PORT)
-        .build()
-        .service();
+    datastore = HELPER.options().toBuilder().namespace("ghijklmnop").build().service();
```

Comment:
@ajkannan I think it would be nice if we overloaded `options` in the helper with a `defaultNamespace`.
Also, we should probably document the namespace setter in the Builder and getter in the options to suggests
that this is the "default namespace".